src/main/java/dev/ftb/mods/ftbessentials/command/TeleportCommands.java

```diff
@@ -89,10 +94,15 @@ public static int rtp(ServerPlayer player) {
 		FTBEPlayerData data = FTBEPlayerData.get(player);
 		return data.rtpTeleporter.teleport(player, p -> {
 			p.displayClientMessage(new TextComponent("Looking for random location..."), false);
-			return findBlockPos(p.server.getLevel(Level.OVERWORLD), p, 1);
+			return findBlockPos(player.getLevel(), p, 1);
 		}).runCommand(player);
 	}
 
+	private static boolean isBlockPosAir(ServerLevel world, BlockPos pos) {
+		BlockState bs = world.getBlockState(pos);
```

Comment:
This should just be `world.getBlockState(pos).isAir()`

Reply:
blockstate .isAir() is deprecated, used .getBlock().isAir(world, state, pos) instead